umad-02-keys-and-authentication.md

```diff
@@ -25,42 +25,56 @@ section.
 
 ## Public Key Exchange
 
-VASPs expose their public keys to other VASPs by responding to `GET` requests at the endpoint
+TThe UMA protocol relies on X.509 certificates for public key exchange among VASPs. In order to obtain an X.509
+certificate, the VASP is required to submit a request to a trusted certificate authority (CA), which issues the
+certificate. The issuing CA must keep track of certificates that are revoked, and provide this information to other
+parties via Certificate Revocation Lists (CRLs) or an Online Certificate Status Protocol (OCSP) server. The URLs for
+accessing a CA's CRL/OCSP can be found inside the certificate, and VASPs should ensure the validity of all of the
+certificates they receive to ensure compliance and security.
+
+It is not recommended for VASPs to use self-signed certificates when communicating with other VASPs outside their
```

Comment:
Because of the above, we can't give a recommendation this strong. All vasps will start with self-signed certs iiuc.